### DISCUSSION TITLE
Unable to remove DRM from Kindle book on Mac

### DISCUSSION THREAD

**Discussion by baqlias** [apprenticeharper/DeDRM_tools#1838] [Q&A] [answered]
[tmpxp2i3ux9.txt](https://github.com/apprenticeharper/DeDRM_tools/files/7139115/tmpxp2i3ux9.txt)

I have a MacBook Air with macOS 11.5.2.  I have Calibre 5.26 and the latest DeDRM plugin installed.  After my Kindle for Mac upgraded itself to 1.33, I went through the process of deregistering it, deleting the appropriate folder in Library, deleting the app, and then installing Kindle for Mac 1.30.  I used Terminal to add the appropriate chmod -x command.  I then downloaded a couple of Kindle books, but Calibre did not remove the DRM.  I've attached the Calibre debug file.  I've tried this a couple of times and can't seem to determine what's wrong with my setup.  Any help is most appreciated!

**Comment by ElleKayEm** [comment, marked as answer]
Mac users have been having trouble with the initial key retrieval with DeDRM 7.2.1.  If you can get things working with calibre 4.23 and DeDRM 6.8.1, then you should be able to return to your current versions.

**Reply by victorsoyvictor** [reply]
This worked for me. I have not returned to current versions though.

**Comment by baqlias** [comment]
That worked!  I am now back to the most current versions of Calibre and DeDRM.  Everything is working fine.  Thank you!

**Reply by ElleKayEm** [reply]
Thanks for reporting back!
